test/zmq_proxy.xrep-xreq.js

```diff
@@ -43,7 +43,7 @@ describe('proxy.xrep-xreq', function() {
       rep.send(msg+' bar');
     });
 
-    setTimeout(function() {
+    setTimeout(function () {
```

Comment:
minor nit, kind of an exception rather than the rule: after anonymous function keywords, no space